### ISSUE TITLE
Add support to pass parameters per axis

### ISSUE THREAD

**Issue by fepegar** [fepegar/torchio#268]
If one has e.g. a stack of microscopy images, rotation around one axis only might be desired.

The interface would be similar to e.g. `Crop`:

```python
import torchio as tio
apply_affine = tio.RandomAffine(degrees=(0, 0, 10))  # -10 to 10 around depth axis
apply_affine = tio.RandomAffine(degrees=(0, 0, 0, 0, -15, 10))  # -15 to 10 around depth axis
apply_affine = tio.RandomAffine(degrees=(0, 0, 0, 0, 10, 10))  # 10 around depth axis
```

**Comment by jxchen01** [comment]
any suggestion on how to implement this? I am very interested in this

**Comment by fepegar** [comment]
Hi, @jxchen01 

I worked a bit on this a while ago, but haven't merged it yet. Given the interest, I'll merge this ASAP, at least for `RandomAffine`.
